README.md

```diff
@@ -801,6 +801,8 @@ The code used by snippets and samples is compiled on the build server. The compi
 
 ### NuGet package references
 
+Docs assumes all dependencies follow SemVer.
```

Comment:
I don't think this is true, or something that logically follows from the guidelines below. I'd just remove this change.